src/crs/projected.jl

```diff
@@ -89,3 +89,6 @@ end
 
 # avoid converting coordinates with the same type as the first argument
 Base.convert(::Type{C}, coords::C) where {Datum,C<:Projected{Datum}} = coords
+
+Base.convert(::Type{Cartesian}, coords::Projected) = convert(Cartesian, convert(LatLon, coords))
```

Comment:
Please move these methods to before the `# projection conversion with same data` methods.